### PR TITLE
Refactoring

### PR DESCRIPTION
**Pull request by pritam-kr**

### CODE REVIEW

**Review by samyakshah3008 — APPROVED** [review]
Good refactoring.
Ready for approval

File: src/Pages/AuthPage/Style.css [hunk 1]
```diff
@@ -16,9 +16,9 @@
 }
 
  
-.gif-wrapper .gif {
+/* .gif-wrapper .gif {
```
Comment (varnita619) [inline src/Pages/AuthPage/Style.css]: remove comments

File: src/Pages/UserProfile/UserProfile.jsx [hunk 2]
```diff
@@ -17,31 +17,42 @@ const UserProfile = () => {
...
+
   //Current Profile
   const currentProfile = users.find(
     (eachUser) => eachUser.username === pathname.username
   );
 
-  useEffect(() => {
-
-    document.title =currentProfile.username
+  // console.log(currentProfile)
```
Comment (varnita619) [inline src/Pages/UserProfile/UserProfile.jsx]: remove console logs

**Review by varnita619 — APPROVED** [review]
Rest all is fine
Code is clean and modular
Ready to merge

**Pull request merged**